src/ralph/cli.py

```diff
@@ -92,23 +92,33 @@ def wrapper(command):
     required=True,
     help="Container format parser used to extract events",
 )
-@click.option(
-    "-c",
-    "--chunksize",
-    type=int,
-    default=DEFAULT_GELF_PARSER_CHUNCK_SIZE,
-    help="Parse events by chunks of size #",
-)
-def extract(parser, chunksize):
+def extract(parser):
     """Extract input events from a container format using a dedicated parser"""
 
-    logger.info(
-        "Extracting events using the %s parser (chunk size: %d)", parser, chunksize
-    )
+    logger.info("Extracting events using the %s parser", parser)
 
     parser = get_class_from_name(parser, PARSERS)()
 
-    for event in parser.parse(sys.stdin, chunksize=chunksize):
+    for event in parser.parse(sys.stdin):
+        click.echo(event)
+
+
+@cli.command()
+@click.option(
+    "-p",
+    "--platform",
+    type=str,
+    required=True,
+    help="The platform (hostname) to use in the xApi statements",
+)
+def convert(platform):
+    """Convert extracted events to xApi"""
+
+    logger.info("Converting events to xAPI")
+
+    converter_selector = XapiConverterSelector(platform)
+
+    for event in converter_selector.convert(sys.stdin):
         click.echo(event)
```

Comment:
Apropos, I tried a version of the converter using `json.dump(event, sys.stout, default=default)` where the default function did the transformation for GetFromField instances instead - but it ended up being slower.
I thought, as json.dump is a C function wrapper it should be faster to let the json library do the transform & writing but unfortunately I was wrong.